README.md

```diff
@@ -188,3 +188,13 @@ export DATABASE_URL=postgres://[user]:[password]@localhost/okapi
 ```sh
 ./script/download_folio_record.rb a123456 > record.json
 ```
+
+#### Synching data
+##### Libraries
+If the name of a library in Folio has changed, you'll want to export the list of libraries with their labels and check it in here.  You can do this using the psql copy command.
+
+```
+\copy (select id, jsonb ->>'code' as code, jsonb->>'name' as name from loclibrary) to 'config/folio/loclibrary.csv' csv header;
```

Comment:
I guess I imagined we'd have a script that stores the full JSON responses like we're doing in Searchworks + sul-requests, or we'd generate something `Traject::TranslationMap` could use natively. Going directly to the database seems unnecessary and would make it harder to wire up to Jenkins (at the very least, we'd need to get the postgres client libraries installed over there 🤷‍♂️ )

Reply:
I've added https://github.com/sul-dlss/searchworks_traject_indexer/pull/1044 to get that data.